### PR TITLE
Fix/misc

### PR DESCRIPTION
**Pull request by PorlyBe**
- fixes jumpy sidebar on mobile (and positions it nicer on desktop)
- upgrade some copy
- fix modal breaking body overflow sometimes
- add a sneaky little indication on tools as to when next available upgrade is unlocked

### CODE REVIEW

File: src/components/Layout/styled.ts [hunk 1]
```diff
@@ -36,5 +36,6 @@ export const ContentsWrapper = styled.div<{ $showMoon?: boolean }>`
     linear-gradient(0deg, #030c14, #030c14);
   background: #030c14;
   min-height: 100%;
-  min-height: 100dvh;
+  min-height: 100vh;
+  min-height: 100svh;
```
Comment (catboop) [inline src/components/Layout/styled.ts:40]: can remove first min-height

**Review by catboop — APPROVED** [review]
is this intentional?
Screenshot 2023-11-09 at 2 48 03 PM

File: src/components/Layout/styled.ts (hunk 1 above)
Comment (PorlyBe) [reply src/components/Layout/styled.ts:40]: no... it's fallback for browsers that dont support svh

**Comment by PorlyBe** [comment]
hmm... not for 0 owed.. will fix

**Pull request merged**